cypress/support/commands.js

```diff
@@ -0,0 +1,90 @@
+// These commands were sourced from the cypress-keycloak-login package
+const utils_1 = require('./utils');
```

Comment:
Any reason we're calling this `utils_1`? Could use destructing here to get only the `decodeToken` function by using `const {.decodeToken } = require('./utils');`.